app/mailers/cancelled_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class CancelledMailer < ApplicationMailer
+  default from: "dontremovethismail@gmail.com"
```

Comment:
Uhm, maybe, it's better to use our official email address? :construction_worker_man: 
By the way, why do you use a separate mailer for cancelled orders?